lib/src/main/resources/com/diffplug/spotless/npm/common-serve.js

```diff
@@ -1,26 +1,26 @@
+import shutdownServer from 'http-graceful-shutdown';
+import express from 'express';
+import fs from 'fs';
+
+//
 // this file will be glued to the top of the specific xy-serve.js file
+//
+
 const debug_serve = false; // set to true for debug log output in node process
-const shutdownServer = require("http-graceful-shutdown");
-const express = require("express");
 const app = express();
 
 app.use(express.json({limit: "50mb"}));
 
-const fs = require("fs");
-
 function debugLog() {
 	if (debug_serve) {
 		console.log.apply(this, arguments)
 	}
 }
 
 function getInstanceId() {
-	const args = process.argv.slice(2);
-
 	// Look for the --node-server-instance-id option
 	let instanceId;
-
-	args.forEach(arg => {
+	process.argv.slice(2).forEach(arg => {
```

Comment:
more cohesion less coupling. now the 2 and 1 is kind of related making it a little less magic.